src/config.ts

```diff
@@ -92,6 +93,7 @@ const mustHave = {
   CONVECTION_API_BASE,
   CONVECTION_APP_ID,
   DELTA_API_BASE,
+  FORCE_URL,
```

Comment:
@erikdstock it may be worth considering if we can put any defaults in place here to have it work as folks would expect. If we have a good way to detect if the running instance is staging vs prod that'd be ideal because then we could default it to `staging.artsy.net` & `artsy.net` respectively.

Reply:
Eh, looking through it I haven't seen a good way to do that. I'm fine w/ skipping it.

Reply:
:+1: https://github.com/artsy/metaphysics/pull/2719/commits/84f4904bef6c2304c7b893ba29b2a09ca0f2c5a4